news/2018-01-20-beatmap-spotlights-december-2017.md

```diff
@@ -0,0 +1,152 @@
+
+## Past Spotlights results
+
+First of all, congratulations to the winners of the November 2017 Spotlights. They receive one month of supporter status for their efforts!
++ **osu!:** [Toy](https://osu.ppy.sh/users/2757689)
++ **osu!taiko:** [iceOC](https://osu.ppy.sh/users/5482401)
++ **osu!catch:** [monstratorfull](https://osu.ppy.sh/users/1646397)
++ **osu!mania:** [\_SkyFall](https://osu.ppy.sh/users/1872276)
+
+# Announcing Spotlights Bounty
+
+I'm glad to announce a new type of competition which will run simultaneous with the monthly other Spotlights. We place a bounty on a theme which lacks mapsets and then we ask you as a mapper to map a mapset and rank it until the deadline. After that we choose the bests from this pool of mapsets and announce it.
+
+## How can you enter?
+
++ Map a mapset for any Loki/Thaehan song. It doesn't matter if its not listed [here](https://osu.ppy.sh/beatmaps/artists/7) feel free to check out his other works at [bandcamp](https://loki2.bandcamp.com/) too.
++ The mapset can be in any mode (osu!, osu!taiko, osu!catch, osu!mania).
++ You must map **at least three** (3) difficulties in an appropriate spread as per the Ranking Criteria, either ENH (easy, normal and hard) or NHI (normal, hard, insane).
++ Use SB01 as a tag for easier find.
++ Do not submit mapsets directly to us, instead ask the Beatmap Nominators for help!
++ Your mapset must be ranked before the 15th of March!
+
+## What are the Prizes?
+
++ **1st place:** 3 months of osu!supporter and a shiny new badge, "Bounty Hunter: Thaehan"
++ **2nd place:** 2 months of osu!supporter
++ **3rd place:** 1 month of osu!supporter
++ The prizes are awarded per game mode
+
+## What are the Prizes?
+
++ **1st place:** 3 months of osu!supporter and 15 Bounty point
++ **2nd place:** 2 months of osu!supporter and 12 Bounty point
++ **3rd place:** 1 month of osu!supporter and 9 Bounty point
++ The points will go down in a respective order
++ At the end of the year these points will decide who will go home with the badge and title "Bounty Hunter 2018: osu!/taiko/catch/mania"
++ The prizes are awarded per game mode
++ All players will be able to earn a shiny new medal "Bounty Hunter: Thaehan".
+
+# December 2017 Spotlights
+
+## osu!
+
+[![](/wiki/shared/news/2018-01-20-beatmap-spotlights-december-2017/impulse.jpg)](https://osu.ppy.sh/beatmapsets/705788)
+
+As one of the most well known mappers in the community, [handsome](https://osu.ppy.sh/users/2123087) continues to impress us with his latest ranked map - Impulse after RELOADED! Featuring 3 specific types of sliders coupled with delightful usage of patterns and small circle size, this map gives a uniquely aim-heavy challenge that fits the song at its finest! Be sure to check out the alluring storyboard made by Darky1 as well.
```

Comment:
Be sure (etc.) by Darky1 as well -> In addition, be sure (etc.) by Darky1
"as well" doesn't sound nice at the end of a sentence